methods.go

```diff
@@ -42,6 +42,23 @@ func (c *Client) GetFreeSpace(path string) (int64, error) {
 	return freeSpace, nil
 }
 
+// GetListenPort return the listen port of the deluge daemon
+func (c *Client) GetListenPort() (uint16, error) {
```

Comment:
Also, this function is missing from the interface.

Reply:
Yes, it have.

Reply:
Yes, it has. and I will check out it for merge from main branch.